project_task_stage_change_restriction/models/project_task.py

```
@@ -0,0 +1,58 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    def _is_move_allowed(self, task, new_stage, user):
```

Ruff (0.11.9)

Missing return type annotation for private function `_is_move_allowed`

Add return type annotation: `bool`

(ANN202)